### PR TITLE
326 muc business hours tests

### PR DESCRIPTION
**Pull request by FabianWilms**
**Description**

Add tests for MucBusinessHours.

**Reference**

Issues #326 

## Summary by CodeRabbit

- **New Features**
	- Introduced a new configuration file for enhanced language server integration, including settings for language, tone, and review workflows.
  
- **Tests**
	- Added a suite of unit tests for the BusinessHoursToggle component, covering rendering, toggling functionality, and computed properties.

### CODE REVIEW

File: src/components/BusinessHours/MucBusinessHours.test.ts [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    });
+
+    it('renders "geschlossen" for days with no opening hours', () => {
+        const wrapper = mount(BusinessHoursToggle, {
+            props: {
+                businessHours,
+            },
+        });
+        const closedDay = businessHours.find(day => day.openingHours.length === 0);
+        expect(wrapper.text()).toEqual(`Öffnungszeiten Mo09:00 - 17:00Di09:00 - 17:00${closedDay.weekDay}geschlossenDo09:00 - 17:00Fr09:00 - 17:00SageschlossenSogeschlossen`);
```
[the remark this answers is not in the document]
Comment (langehm) [reply src/components/BusinessHours/MucBusinessHours.test.ts:102]: Should be more robust this way

**Review by langehm — APPROVED** [review]
Chech the last comment of rabbitai - this could be benefitial

File: src/components/BusinessHours/MucBusinessHours.test.ts [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        businessHours,
+        toggleable: true,
+      },
+      slots: {
+        hint: () => h("div", "This is a hint"),
+      },
+    });
+    expect(wrapper.find(".hint").exists()).toBe(true);
+    expect(wrapper.find(".hint").text()).toBe("This is a hint");
+  });
```
[the remark this answers is not in the document]
Comment (FabianWilms) [reply src/components/BusinessHours/MucBusinessHours.test.ts:143]: this component does currently not need to be accessible. these tests can be added once this requirement arises.

**Review by langehm — APPROVED** [review]
lgtm.
i dont know much about the config but it works in this issue nicely

**Pull request merged**